godot-core/src/builtin/dictionary.rs

```diff
@@ -426,7 +428,9 @@ impl<'a> DictionaryIter<'a> {
         } else {
             Self::call_next(self.dictionary, self.last_key.take()?)
         };
-
+        if self.next_idx < self.dictionary.len() {
+            self.next_idx += 1;
+        }
```

Comment:
Please use empty lines before/after blocks:
```suggestion

        if self.next_idx < self.dictionary.len() {
            self.next_idx += 1;
        }
        
```

Reply:
Added spacing in recommended places to improve visibility.